packages/arb-token-bridge-ui/tests/e2e/specs/readClassicDeposits.cy.ts

```diff
@@ -69,45 +64,35 @@ describe('Read classic deposit messages', () => {
         ])
       )
 
-      cy.changeMetamaskNetwork('mainnet').then(() => {
-        cy.openTransactionsPanel()
-
-        const l1TxHash =
-          '0x00000a813d47f2c478dcc3298d5361cb3aed817648f25cace6d0c1a59d2b8309'
-        const l2TxHash =
-          '0xd3ff2a70a115411e1ae4917351dca49281368684394d0dcac136fa08d9d9b436'
-
-        cy.findByLabelText(/l1 transaction status/i).should(
-          'contain',
-          'Success'
-        )
-        cy.findByLabelText(/l2 transaction status/i).should(
-          'contain',
-          'Success'
-        )
-
-        cy.findByLabelText(/l1 transaction link/i).should(
-          'contain',
-          `${shortenTxHash(l1TxHash)}`
-        )
-
-        cy.findByLabelText(/l2 transaction link/i).should(
-          'contain',
-          `${shortenTxHash(l2TxHash)}`
-        )
-      })
+      cy.openTransactionsPanel()
+
+      const l1TxHash =
+        '0x00000a813d47f2c478dcc3298d5361cb3aed817648f25cace6d0c1a59d2b8309'
+      const l2TxHash =
+        '0xd3ff2a70a115411e1ae4917351dca49281368684394d0dcac136fa08d9d9b436'
+
+      cy.findByLabelText(/l1 transaction status/i).should('contain', 'Success')
+      cy.findByLabelText(/l2 transaction status/i).should('contain', 'Success')
+
+      cy.findByLabelText(/l1 transaction link/i).should(
+        'contain',
+        `${shortenTxHash(l1TxHash)}`
+      )
+
+      cy.findByLabelText(/l2 transaction link/i).should(
+        'contain',
+        `${shortenTxHash(l2TxHash)}`
+      )
     })
   })
 
   context('User has classic ERC-20 deposit transaction', () => {
     // log in to metamask
     before(() => {
-      cy.login({ networkType: 'L1' })
-    })
-
-    after(() => {
-      // after all assertions are executed, logout and reset the account
-      cy.logout()
+      cy.login({
```

Comment:
Let's remove it. It's better if it's in `beforeEach`

Reply:
Explained above